modules/tinymce/src/core/main/ts/api/EditorOptions.ts

```diff
@@ -181,6 +181,8 @@ const getBuiltInProcessor = <K extends BuiltInOptionType>(type: K): Processor<Bu
         return (val) => Type.isArrayOf(val, Type.isObject);
       case 'regexp':
         return (val) => Type.is(val, RegExp);
+      default:
+        return Fun.always;
```

Comment:
Not sure why this is needed, as every case was covered by the `case` statements above and I double checked a couple times. Either way, this seems like a reasonable default.